js-exercises/05-js-conditionals/index.js

```diff
@@ -0,0 +1,11 @@
+"use strict"
+alert("Enter two numbers to see if the first is greater than the second");
+let firstValue = prompt( "First value", 100)
+let secondValue = prompt( "Second value", 100)
+if(Number(firstValue) > Number(secondValue)){
```

Comment:
It would be better to do the conversion once.
`let firstValue = +prompt( "First value", 100)`
`let secondValue = +prompt( "Second value", 100)`